presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -117,150 +142,73 @@ filesystem directory as a Hive Metastore. See :ref:`installation/deployment:File
-======================================================== ============================================================ ============
-.. _constructor: https://github.com/apache/hadoop/blob/02a9190af5f8264e25966a80c8f9ea9bb6677899/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java#L844-L875
-
-Avro Configuration Properties
------------------------------
-
-When querying or creating Avro-formatted tables with the Hive connector, you may need to supply or override the Avro schema. In addition, Hive Metastore, especially Hive 3.x, must be configured to read storage schemas for Avro tables.
-
-Table Properties
-^^^^^^^^^^^^^^^^
+``hive.non-managed-table-writes-enabled``          Enable writes to non-managed (external) Hive tables.         ``false``
 
-These properties can be used when creating or querying Avro tables in Presto:
+``hive.non-managed-table-creates-enabled``         Enable creating non-managed (external) Hive tables.          ``true``
 
-======================================================== ============================================================================== ======================================================================================
-Property Name                                            Description                                                                    Default
-======================================================== ============================================================================== ======================================================================================
-``avro_schema_url``                                      URL or path (HDFS, S3, HTTP, or others) to the Avro schema file for             None (must be specified if Metastore does not provide or you need to
-                                                         reading an Avro-formatted table. If specified, Presto will fetch                override schema)
-                                                         and use this schema instead of relying on any schema in the
-                                                         Metastore.
-======================================================== ============================================================================== ======================================================================================
+``hive.collect-column-statistics-on-write``        Enables automatic column level statistics collection         ``false``
+                                                   on write. See `Table Statistics <#table-statistics>`__ for
+                                                   details.
 
-Hive Metastore Configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``hive.s3select-pushdown.enabled``                 Enable query pushdown to AWS S3 Select service.              ``false``
 
-To support Avro tables with schema properties when using Hive 3.x, you must configure the Hive Metastore service:
```

Comment:
I see some Avro related changes deleted as well. Are we removing them completely?